.github/workflows/fakedroid-odt.yml

```diff
@@ -30,5 +30,5 @@ jobs:
         tests/fakedroid.sh nix-on-droid on-device-test
 
     - name: upload
-      if: always()
+      if: always() && github.event_name != 'pull_request'
```

Comment:
Hope it can be shortened to just `github.event_name != 'pull_request'`.

Reply:
Good question, I am not sure, I will test it with this PR.

Reply:
`always()` does more magic than just returning `true`: https://github.com/t184256/nix-on-droid/runs/4526292325?check_suite_focus=true

But `continue-on-error` would work: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

Reply:
Yeah, I take my suggestion back O.o